distributions/openhab-offline/pom.xml

```diff
@@ -105,6 +105,7 @@
                 <configuration>
                     <installedFeatures>
                         <feature>wrapper</feature>
+                        <feature>management</feature>
```

Comment:
This way, the jars are included in the offline install but not activated, so that you can still do 'feature:install management' to install it offline.